src/kernel/task.zig

```diff
@@ -178,43 +145,43 @@ test "init out of memory for stack" {
     expectEqual(all_pids.bitmap, 1);
 }
 
-test "init expected setup" {
-    var task = try Task.init(test_fn1, std.testing.allocator);
+test "create expected setup" {
```

Comment:
Hmm I think this should be moved back to arch.zig since it depends on the implementation there.